src/ARequestHandler.cpp

```diff
@@ -0,0 +1,22 @@
+#include "ARequestHandler.hpp"
```

Comment:
I know I did it. But being a abstract class I guess it would be more correct to have the implementation of the constructor and stuff directly in the header. Can you check this? Eventually I can do it, since I created the .cpp file to resolve the compilation issue on my machine.

Reply:
As I wrote in https://github.com/dantol29/webserver/pull/117/files/19b7efdf5c56ab1f0617af11a3ffb949d4f90b95#r1568712563 , I don't see at the moment why we have `ARequestHandler() ` altogether because it is used only for the CGI, can't we just have the CGIHandler ?

Reply:
Because we have a StaticRequestHanlder object, which serves static files? How are we serving static files?